tests/lint/git-black.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+set -e
+set -u
+set -o pipefail
+
+if [[ "$1" == "-i" ]]; then
+    INPLACE_FORMAT=1
+    shift 1
+else
+    INPLACE_FORMAT=0
+fi
+
+if [[ "$#" -lt 1 ]]; then
+    echo "Usage: tests/lint/git-black.sh [-i] <commit>"
+    echo ""
+    echo "Run black-format on files that changed since <commit>"
+    echo "Examples:"
+    echo "- Compare last one commit: tests/lint/git-black.sh HEAD~1"
+    echo "- Compare against upstream/master: tests/lint/git-black.sh upstream/master"
+    echo "You can also add -i option to do inplace format"
+    exit 1
+fi
+
+cleanup()
+{
+  rm -rf /tmp/$$.black-format.txt
+}
+trap cleanup 0
+
+
+if [ -x "$(command -v black)" ]; then
+    BLACK=black
+else
+    echo "Cannot find black"
+    exit 1
+fi
+
+# Print out specific version
+
+echo "Version Information: $(${BLACK} --version)"
```

Comment:
Seems like `BLACK` is only used for displaying the version. If so, how about move this line to the if-block so that we can get rid of `BLACK` at all.